gordon-plugin/src/main/kotlin/com/banno/gordon/AdbExtensions.kt

```diff
@@ -43,7 +43,7 @@ internal fun JadbDevice.executeShellWithTimeout(
 internal fun JadbDevice.isTablet(): IO<Boolean> =
     executeShellWithTimeout(20_000, "getprop", "ro.build.characteristics").map { it!!.contains("tablet") }
 
-internal fun JadbDevice.installApk(apk: File, vararg options: String) = fx {
+internal fun JadbDevice.installApk(apk: File, vararg options: String) = bindingConcurrent {
```

Comment:
```suggestion
internal fun JadbDevice.installApk(apk: File, vararg options: String) = IO.fx {
```

Using `IO.fx` or just `fx` here better matches the Arrow docs. 

I'm not sure if `bindingConcurrent` is equivalent judging from the source.